make/autoconf/flags-ldflags.m4

```diff
@@ -21,10 +21,13 @@
 # Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
 # or visit www.oracle.com if you need additional information or have any
 # questions.
-#
 
 ################################################################################
 #
+# ===========================================================================
+# (c) Copyright IBM Corp. 2024, 2024 All Rights Reserved
+# ===========================================================================
+#
```

Comment:
Please follow the pattern elsewhere (e.g. jdk17 - a blank line, then lines 27-29, not line 30).
```
################################################################################
#

# ===========================================================================
# (c) Copyright IBM Corp. 2024, 2024 All Rights Reserved
# ===========================================================================

AC_DEFUN([FLAGS_SETUP_LDFLAGS],
```